R/utils.R

```diff
@@ -1536,3 +1536,38 @@ validate_keys <- function(string, is_template) {
     return(string)
   }
 }
+
+# Dash2 Helper Functions
+
+#' Is the given object a Dash app?
+#' @param x Any object.
+is_dash_app <- function(x) {
+  inherits(x, "Dash")
+}
+
+assert_dash <- function(x) {
+  if (!is_dash_app(x)) {
+    stop("You must provide a Dash app object (created with `dash::Dash$new()` or `dash2::dash_app()`)", call. = FALSE)
```

Comment:
Still a bunch of references to `dash2` here and in `wrappers.R` (only the link to https://github.com/daattali/dash2 should stay)

Reply:
Fixed in ebdbe37